src/redis-cli.c

```diff
@@ -2008,9 +2011,12 @@ static int issueCommandRepeat(int argc, char **argv, long repeat) {
                 return REDIS_ERR;
             }
         }
+
         /* Issue the command again if we got redirected in cluster mode */
         if (config.cluster_mode && config.cluster_reissue_command) {
-            cliConnect(CC_FORCE);
+            /* If cliConnect fails, sleep for a while and try again. */
+            if (cliConnect(CC_FORCE) != REDIS_OK)
+                sleep(1);
```

Comment:
well, i guess there's always room for improvement..
before this fix it would have had an infinite busy loop (possibly flooding the console with prints).
now we at least throttled the prints, right?

looking further i now see the screenshot at the top mentions `(will sleep 1 second)` but i can't find that in the code.
what am i missing?

Reply:
@oranagra oh. `(will sleep 1 second)` That one is a debug print i added is before. Just for test(good looking)

Also I improve the error handing. Added some log. It is ok with?

Reply:
@oranagra Yes. In the above example. It will have an endless busy loop (if we don't sleep. will flood the console with prints).  
My first screenshot the log i added i think it is in `context = redisConnect(config.hostip,config.hostport);` position
The sleep i think it has been handled in method `issueCommandRepeat`